.travis.yml

```diff
@@ -1,5 +1,12 @@
 language: python
 
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - xz-lzma
```

Comment:
This doesn't seem to be necessary as it is already installed. However, it seems important to state this explicitly.